### ISSUE TITLE
coconut -w errors with "NameError: name 'args' is not defined"

### ISSUE THREAD

**Issue by ivanmkc** [evhub/coconut#638]
Command: `python -m coconut . -wk`
Error when file is changed:
```
  File "/Users/chosan/code/converso/dictionaries/env/lib/python3.8/site-packages/watchdog/observers/api.py", line 199, in run
    self.dispatch_events(self.event_queue, self.timeout)
  File "/Users/chosan/code/converso/dictionaries/env/lib/python3.8/site-packages/watchdog/observers/api.py", line 372, in dispatch_events
    handler.dispatch(event)
  File "/Users/chosan/code/converso/dictionaries/env/lib/python3.8/site-packages/watchdog/events.py", line 272, in dispatch
    {
  File "/Users/chosan/code/converso/dictionaries/env/lib/python3.8/site-packages/coconut/command/watch.py", line 60, in on_modified
    self.recompile(path, *args, **kwargs)
NameError: name 'args' is not defined
`````````

**Comment by evhub** [comment]
@ivanmkc Fixed on `coconut-develop>=2.0.0-a_dev29` (`pip install -U coconut-develop>=2.0.0-a_dev29` to get the fix).

**Comment by ivanmkc** [comment]
Thanks for the quick fix.
